src/less/variables/ds6/base64-variables.less

```diff
@@ -93,7 +93,7 @@
 @icon-profile-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLXByb2ZpbGUiIHZpZXdCb3g9IjAgMCAyMiAyNCI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTExIDBoLS4wMDFhNiA2IDAgMDAtNiA2YzAgMS44MDguODAzIDUuNDI0IDIuMDY3IDYuNTI0IDAgMC0zLjM0NSAxLjM4OC00LjM5MyAxLjk4MkMuOTAxIDE1LjUxMS0uMDAyIDE3LjYyOSAwIDE5LjcyOHYzLjIyNkMwIDIzLjUzMi40NDcgMjQgMSAyNGgxOS45OTljLjU1MyAwIDEtLjQ2OCAxLTEuMDQ2di0zLjIyNmMuMDAyLTIuMDk5LS45MDEtNC4yMTctMi42NzMtNS4yMjItMS4wNDgtLjU5NC00LjM5My0xLjk4Mi00LjM5My0xLjk4MkMxNi4xOTcgMTEuNDI0IDE3IDcuODA4IDE3IDZhNiA2IDAgMDAtNi02bTAgMmMyLjIwNSAwIDQgMS43OTQgNCA0IDAgMS43NDctLjgzNiA0LjUyMy0xLjM4IDUuMDE2YTIgMiAwIDAwLjU0NiAzLjM1NWMxLjI1LjUxOSAzLjQ4OCAxLjQ4NiA0LjE3NCAxLjg3NSAxLjAxLjU3MiAxLjY2MSAxLjkzOCAxLjY1OSAzLjQ4MlYyMkgydi0yLjI3NGMtLjAwMi0xLjU0Mi42NDktMi45MDggMS42NTktMy40OC42ODYtLjM4OSAyLjkyNC0xLjM1NiA0LjE3NC0xLjg3NWEyLjAwNCAyLjAwNCAwIDAwMS4xOS0xLjUxOSAyLjAzMiAyLjAzMiAwIDAwLS42NDMtMS44MzZDNy44MzQgMTAuNTIzIDYuOTk5IDcuNzQ3IDYuOTk5IDZjMC0yLjIwNiAxLjc5NS00IDQuMDAxLTQiIGZpbGw9IiMxMTE4MjAiLz4KPC9zdmc+";
 @icon-clock-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLWNsb2NrIiB2aWV3Qm94PSIwIDAgMjIgMjIiPgo8cGF0aCBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0yMiAxMWMwIDYuMDc1LTQuOTI1IDExLTExIDExUzAgMTcuMDc1IDAgMTEgNC45MjUgMCAxMSAwczExIDQuOTI1IDExIDExem0tMiAwYTkgOSAwIDAxLTkgOSA5IDkgMCAwMS05LTkgOSA5IDAgMDE5LTkgOSA5IDAgMDE5IDl6TTEwIDVhMSAxIDAgMDEyIDB2NS41OWwyLjcgMi43MWExIDEgMCAwMS0xLjQxIDEuNDFsLTMtM0ExIDEgMCAwMTEwIDExVjV6IiBmaWxsPSIjMTExODIwIi8+Cjwvc3ZnPg==";
-@icon-radio-checked-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLXJhZGlvLWNoZWNrZWQiIHZpZXdCb3g9IjAgMCAyNCAyNCI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTEyIDBDNS4zNzMgMCAwIDUuMzczIDAgMTJzNS4zNzMgMTIgMTIgMTIgMTItNS4zNzMgMTItMTJBMTEuOTk4IDExLjk5OCAwIDAwMTIgMHptMCAyMkM2LjQ3NyAyMiAyIDE3LjUyMyAyIDEyUzYuNDc3IDIgMTIgMnMxMCA0LjQ3NyAxMCAxMC00LjQ3NyAxMC0xMCAxMHptNy0xMGE3IDcgMCAxMS0xNCAwIDcgNyAwIDAxMTQgMHoiIGZpbGw9IiMxMTE4MjAiLz4KPC9zdmc+";
+@icon-radio-checked-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLXJhZGlvLWNoZWNrZWQiIHZpZXdCb3g9IjAgMCAyNCAyNCI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTEyIDBDNS4zNzMgMCAwIDUuMzczIDAgMTJzNS4zNzMgMTIgMTIgMTIgMTItNS4zNzMgMTItMTJBMTEuOTk4IDExLjk5OCAwIDAwMTIgMHptMCAyMkM2LjQ3NyAyMiAyIDE3LjUyMyAyIDEyUzYuNDc3IDIgMTIgMnMxMCA0LjQ3NyAxMCAxMC00LjQ3NyAxMC0xMCAxMHptNy0xMGE3IDcgMCAxMS0xNCAwIDcgNyAwIDAxMTQgMHoiIGZpbGw9IiMzNjY1RjIiLz4KPC9zdmc+";
```

Comment:
The blue color is one digit off. You have `#3665F2`, but it's `#3665F3` in our [color variable](https://github.com/eBay/skin/blob/9.2.0/dist/variables/ds6/color-variables.less#L42).